.circleci/config.yml

```diff
@@ -507,6 +507,7 @@ jobs:
               # run tests with code coverage
               CORRADE_TEST_COLOR=ON GTEST_COLOR=yes ./build.sh --headless \
                 --bullet \
+                --vhacd \
```

Comment:
You also need to add it above so that when clang-tidy is run, it is also built with VHACD so that the interface gets evaluated by clang-tidy.